Fix Portuguese flag

BChapoulie opened the pull request:
There is a slight graphical issue in the Portuguese flag where two lines are not perfectly aligned:

*Before*
![image](https://github.com/lipis/flag-icons/assets/20483989/ce2ca67c-b046-44d9-b055-fbd044cea618)

*After*
![image](https://github.com/lipis/flag-icons/assets/20483989/14f8fb1a-89b8-49b7-94e0-e03c35e84419)

PriitUring commented:
Maybe better title: "Fix Portuguese flag", then more consistent with other pull requests.

You could also ammend commit and squash commits, because most already merged pull request commits are not seperated with individual 1x1 and 4x3 commits.

https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits-in-github-desktop

BChapoulie commented:
I renamed the PR and squashed the commit (sorry for that, I am used to work with Azure devops at my job and the commit squash is part of the PR merging policy so I thought it might be the same on Github 😅)
Does it look better to you now?

lipis commented:
Thank you!

lipis commented:
I'm squashing anyways all PRs so it doesn't matter!
